db/migrate/20160406110329_create_flood_risk_engine_addresses.rb

```diff
@@ -18,6 +18,6 @@ def change
       t.timestamps null: false
     end
 
-    add_foreign_key :flood_risk_engine_addresses, :flood_risk_engine_contacts
+
```

Comment:
@woodpigeon No it was only moved to later migration (! hope). Postgres would not create it as the contact table was not yet present


 +    add_foreign_key :flood_risk_engine_addresses, :flood_risk_engine_contacts, column: :contact_id